chart/values.yaml

```diff
@@ -71,7 +83,9 @@ controller:
       pullSecrets: []
 
     csiResizer:
-      name: registry.k8s.io/sig-storage/csi-resizer:v1.7.0
+      name: ""
+      repository: registry.k8s.io/sig-storage/csi-resizer
+      tag: v1.7.0
```

Comment:
Do you think it makes sens to reuse the `name` value, and only append the tag, if a `tag` value is present? This way we don't use the `repository` value, that I find a bit confusing.

Reply:
Yes, reusing name for the registry + repository sounds like a good idea. Then this PR will only separate the tag from the image name.